src/parameters/all_parameters.h

```diff
@@ -175,11 +175,13 @@ class AllParameters
         dual_weighted_residual_mesh_adaptation,
         taylor_green_vortex_energy_check,
         taylor_green_vortex_restart_check,
+        homogeneous_isotropic_turbulence_initialization_check,
         time_refinement_study,
         time_refinement_study_reference,
         burgers_energy_conservation_rrk,
         euler_entropy_conserving_split_forms_check,
-        homogeneous_isotropic_turbulence_initialization_check,
```

Comment:
maybe revert this change so we all have 1 less merge conflict to resolve

Reply:
Sure, have reverted the ordering change